simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -111,8 +111,27 @@ private static String getHelpMessage(String metricName, Metric metric){
                         Arrays.asList(new MetricFamilySamples.Sample(name + "_total",
                                 new ArrayList<String>(),
                                 new ArrayList<String>(),
-                                meter.getCount())))
-
+                                meter.getCount()))),
+                new MetricFamilySamples(name + "_one_minute_rate", Type.COUNTER, getHelpMessage(dropwizardName, meter),
```

Comment:
Counters only go up. So you would want a `Gauge` instead.

That being said, Prometheus calculates rates better based off the `_total` that are already reported. So I would recommend against adding this feature.